doc/man1/flux-hostlist.rst

```diff
@@ -0,0 +1,198 @@
+.. flux-help-section: other
+
+================
+flux-hostlist(1)
+================
```

Comment:
perhaps should document exit value?  I noticed in a test or two of exit 1 vs 0, wasn't sure what the rules are (haven't looked at tool code yet)

Reply:
Oh good suggestion.